mofiebiger/ember-trivia
# Ember trivia start quiz

# ✨ What Changed & Why

**NOTE: Too much was done in this PR - next time make PRs smaller & more specific!!!!**

- **Add Colour Variables:** This is done to organise colours and be more explicit. Not necessary, but will most likely be done in a production environment.

- **Alphabetise CSS:** This is done to easily navigate styles and create a global pattern. Not necessary, but will most likely be done in a production environment.

- **Created start-game route:** This is the first page all users will see. It does not have a _Modal_ and contains the `<StartGamePage />` component within it. In the future the dropdown content will be passed to a Model and transition to a new route.

- **Created `<StartGamePage />` component:** This component contains the contents of the start-game route. Inside this component is the `<DisplayBox />` component which holds two `ember-power-select` drop downs and a `LinkTo` button.

- **Created `<DisplayBox />` component:** This is a generic component and will be used on all pages of the app in the future. It is very generic, only containing a header and a  body. Text can only be added into the header, but anything can be added to the body. 

- **Update translations:** New text was added! There are not going to be multiple languages in this app, but most SASS companies will want their applications in many languages. 

- **Update style to header / footer:** The big changes here are updating the logo, adding `drop-shadow` to both components, and making images larger.

- **Add `ember-power-select`:** Embers version of a dropdown. We use this, because much of the functionality is done for us!

- **Add `ember-truth-helpers`:** This allows us to use logic within templates! Wooooo!

# 👹 Before

<img width="1680" alt="Screenshot 2020-05-14 at 17 41 54" src="https://user-images.githubusercontent.com/43208685/81961684-9ca32b80-960a-11ea-984b-5e5c686392cc.png">

# 😇 After

![startgamepage](https://user-images.githubusercontent.com/43208685/86483765-a57fc600-bd4c-11ea-9d25-b2e451c24fe1.gif)

# 🌈 Testing

1. Visit [Ember-Trivia-Website](https://5eff52e34792090007046463--kind-haibt-4424bd.netlify.app/)
1. Notice header & footer have been updated
1. Observe a model is centred with two dropdowns
